CHANGELOG.adoc

```diff
@@ -14,6 +14,7 @@
 
 - https://github.com/eclipse-syson/syson/issues/1028[#1028] [import] Improve `SuccessionAsUsage` with implicit source feature.
 - https://github.com/eclipse-syson/syson/issues/1030[#1030] `ConnectorAsUsage.getSourceFeature` and `ConnectorAsUsage.getTargetFeature` should redefine `Relationship.source` and `Relationship.target` features
+- https://github.com/eclipse-syson/syson/issues/1042[#1042] Improve SuccessionAsUsage defining directly the target actions.
```

Comment:
Please do not forget to add a category (here it is [import]).
SuccessionAsUsage => `SuccessionAsUsage` (please surround with \`)

Reply:
Done.